src/styles/_catalog.scss

```diff
@@ -1,22 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  grid-template-rows: repeat(auto-fill, $heightCard);
```

Comment:
Just a suggestion, no need to fix. 
You can specify 1 column instead of auto-fill as this is your default value and delete "1px" 1 from $media collection.